app/src/main/res/values/strings.xml

```diff
@@ -21,10 +21,15 @@
   <string name="main_refresh_error">Error refreshing files</string>
   <string name="main_username">username</string>
   <string name="main_feedback">Send us feedback</string>
+  <string name="main_faq">FAQ</string>
   <string name="main_feedback_email_subject">Envelop feedback</string>
   <string name="logout">Logout</string>
   <string name="logout_confirm">Are you sure?</string>
 
+  <!-- FAQ -->
+  <string name="faq_title">FAQS</string>
+  <string name="faq_host_not_found"> The web page could not be loaded. Check your internet connection </string>
```

Comment:
Maybe write something a bit more specific like:
`The FAQs web page could not be loaded. Please check your connection and try again.`